daemon/algod/api/algod.oas2.json

```diff
@@ -1419,6 +1419,64 @@
         }
       }
     },
+    "/v2/teal/disassemble": {
+      "post": {
+        "description": "Given the base64 encoded program bytes, return the TEAL source code in plain text. This endpoint is only enabled when a node's configuration file sets EnableDeveloperAPI to true.",
+        "consumes": [
+          "text/plain"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "schemes": [
+          "http"
+        ],
+        "summary": "Disassemble program bytes into the TEAL source code.",
+        "operationId": "TealDisassemble",
+        "parameters": [
+          {
+            "description": "TEAL program binary to be disassembled",
```

Comment:
The current implementation requires the program to be base64 encoded. If you want to keep that approach, I'd recommend specifying that here.

Though, since HTTP POST request bodies are capable of accepting binary content, why not accept the binary program directly? There is no technical requirement for base64 encoding in this case, unlike when the program is inserted into a JSON object.

Reply:
> Though, since HTTP POST request bodies are capable of accepting binary content, why not accept the binary program directly? 

It's a good point, I initially settled on base64 to keep parity between the `compile` endpoint result. I can see an argument for taking the binary too, since it could save everyone one encoding, and if you think it's a much better option, we could change the API definition to take an `array` of `uint8` instead.

Reply:
I'd be in favor of having this endpoint accept the binary program. If you want to make that change, I'd suggest looking at the `/v2/transactions` endpoint to see how it defines its binary input.

Reply:
Great - I updated the API specs to reflect this change to take in a binary